### PR TITLE
React Version

### PR DESCRIPTION
**Pull request by farskid**
Added React component with respect to the Video Player API

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,6 +76,34 @@ new InitPxVideo({
...
+    lang: 'EN',
+    default: true
+  }}
+  poster="media/poster_PayPal_Austin2.jpg"
+  width="640"
+  height="360"
+  controls={true}
+  id="myvid"
+  fallback={true}
+	seekInterval={20}
```
Comment (hozefaj) [inline README.md]: align this with other attributes please

**Comment by farskid** [comment]
@hozefaj Updated Readme

**Comment by hozefaj** [comment]
@farskid  Also can you `rebase` your branch from `master`. Not sure why the extra commits from @weboverhauls are a part of this PR. Also would make sense to squash the commits.

**Comment by farskid** [comment]
@hozefaj About the rebase and squash: Agree.
Also not sure why those commits came around the PR!

**Comment by farskid** [comment]
@hozefaj Quick question: The repo and yet `master` branch have not been updated for almost 8 days, So there are no needs for a rebase. I guess a simple squash would do the trick, right?

**Comment by hozefaj** [comment]
Yes that should work. I was thinking that reading might fix where other commits are there a part of the commit.

**Comment by farskid** [comment]
@hozefaj OK I guess the branch is screwed up with squashing. wouldn't it just be easier if you could squash the commits before merging? Github now allows this. BTW, manually squashing by me would require a force push onto branch react which has created conflicts.

**Comment by hozefaj** [comment]
First lets finish the pending items. Then figure out how to come out of this mess.

`force` pushing on the branch is fine in this case.

**Comment by farskid** [comment]
@hozefaj Alright. I'll send a new PR with neat history and let you know about closing this one.

**Pull request closed without merging**